api/api.go

```diff
@@ -47,7 +49,7 @@ func CreateDatasetAPI(host string, secretKey string, router *mux.Router, dataSto
 	api.router.HandleFunc("/instances/{id}/events", api.privateAuth.Check(instance.AddEvent)).Methods("POST")
 	api.router.HandleFunc("/instances/{id}/inserted_observations/{inserted_observations}", api.privateAuth.Check(instance.UpdateObservations)).Methods("PUT")
 
-	dimension := dimension.Store{api.dataStore.Backend}
+	dimension := dimensioncache.Store{api.dataStore.Backend}
 	api.router.HandleFunc("/instances/{id}/dimensions", dimension.GetNodes).Methods("GET")
 	api.router.HandleFunc("/instances/{id}/dimensions/{dimension}/options", dimension.GetUnique).Methods("GET")
 	api.router.HandleFunc("/instances/{id}/dimensions/{dimension}/options/{value}", api.privateAuth.Check(dimension.Add)).Methods("PUT")
```

Comment:
The dimensions we store during the import process are only used for caching purposes, the front end is using the code-list api to get the dimension options. This is why I wanted to have cache in the name.

Reply:
I think there's a middle ground. It's still useful to have DimensionCache or CachedDimension to separate it from the Dimension objects returned on the public API - but within the CachedDimension, where we have value, we should call it option as that is what it will become.